src/content-helper/common/content-helper-error.tsx

```diff
@@ -87,23 +103,22 @@ export class ContentHelperError extends Error {
 				'The Parse.ly API did not respond in a timely manner. Please try again later.',
 				'wp-parsely'
 			);
-		} else if ( this.code === ContentHelperErrorCode.ParselySchemaValidationFailed ) {
+		} else if ( this.code === ContentHelperErrorCode.ParselySuggestionsApiSchemaError ) {
 			this.message = __(
 				'The Parse.ly API returned a validation error. Please try again later.',
```

Comment:
Arthur has suggested that for this error code, it doesn't make sense to tell the users to retry later, since it will hit the same error. Perhaps we can change to something like "Please try again with different parameters"?